README.md

```diff
@@ -1,19 +1,33 @@
-# .NET Core Samples
+# .NET Samples
 
-This folder contains all the sample code that is part of any topic under
-the .NET Core documentation. There are several different projects that
+This repo contains all the sample code that is part of any topic under
+the .NET documentation. There are several different projects that
 are organized in sub-folders. These sub-folders are organized similar
-to the organization of the docs for .NET Core.
+to the organization of the docs for .NET.
 
 Some of the articles will have more than one sample associated with them. 
 
 The readme.md file for each sample will refer to the article so that
```

Comment:
nit: should this be moved to line 16 where you specifically talk about samples? snippets won't have readme file.